th2_data_services/data_source/lwdp/commands/http.py

```diff
@@ -1446,7 +1497,13 @@ def __init__(
             streams: List of streams to search messages from the specified groups.
                 You will receive only the specified streams and directions for them.
                 You can specify direction for your streams.
-                e.g. ['stream_abc:1']. 1 - IN, 2 - OUT.
```

Comment:
Hi

You have broken backward compatibility by this change.
--

1) I added `T_streams` variable that describes expected streams format. See line 63.

You need to change code, so it fits this format.
So streams can be
- just str e.g. "stream" or "stream:2"
- dict -- a new format that you used to replace previous approach
- Stream or Streams objects
- list of these variables


2) Due to a new dict format was introduced, you have to make sure, that Stream and Streams return correct values if we provide them to `post_download_messages` (expects dicts). 
Probably we can just add additional method to `Stream and Streams ` classes